tests/components/mill/test_climate.py

```diff
@@ -0,0 +1,652 @@
+"""Tests for Mill climate."""
+
+from unittest.mock import patch
+
+from mill import Heater
+from mill_local import OperationMode
+import pytest
+
+from homeassistant.components import mill
+from homeassistant.components.climate import (
+    ATTR_HVAC_MODE,
+    DOMAIN as CLIMATE_DOMAIN,
+    SERVICE_SET_HVAC_MODE,
+    SERVICE_SET_TEMPERATURE,
+    HVACMode,
+)
+from homeassistant.components.mill.const import DOMAIN
+from homeassistant.const import ATTR_ENTITY_ID, ATTR_TEMPERATURE, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import HomeAssistantError
+
+from tests.common import MockConfigEntry
+
+HEATER_ID = "dev_id"
+HEATER_NAME = "heater_name"
+ENTITY_CLIMATE = f"climate.{HEATER_NAME}"
+
+## MILL AND LOCAL MILL FIXTURES
+
+
+@pytest.fixture
+async def mock_mill():
+    """Mock the mill.Mill object.
+
+    It is imported and initialized only in /homeassistant/components/mill/__init__.py
+    """
+
+    with (
+        patch(
+            "homeassistant.components.mill.Mill",
+            autospec=True,
+        ) as mock_mill_class,
+        # disable recorder behaviour
+        patch(
+            "homeassistant.components.mill.coordinator.MillHistoricDataUpdateCoordinator._async_update_data",
```

Comment:
Please don't patch anything on the coordinator. What are we aiming to do with this patch?